py3status/modules/keyboard_locks.py

```diff
@@ -7,8 +7,8 @@
     format: display format for this module (default '{caps} {num} {scr}')
     icon_caps_off: show when Caps Lock is off (default 'CAPS')
     icon_caps_on: show when Caps Lock is on (default 'CAPS')
-    icon_num_off: show when Num Lock is off (default 'NUM')
-    icon_num_on: show when Num Lock is off (default 'NUM')
+    icon_num_off: show when Numeric Lock is off (default 'NUM')
+    icon_num_on: show when Numeric Lock is on (default 'NUM')
```

Comment:
`Numeric lock` sounds horrible to me can we keep `num lock` which is what everyone calls it.